### PR TITLE
PR to merge Feature into development

### PR DESCRIPTION
**Pull request by UmairAhmad125**
In this project, I created a minimalist restaurant page that contains.
- A landing page that welcomes users.
- A menu page that displays the menu.
- A contact us page that contains a simple contact form.

### CODE REVIEW

File: src/js/navbar.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  tabsCont.appendChild(contact);
+
+  navCont.classList.add('navCont');
+  const containerOut = document.getElementById('content');
+  containerOut.innerHTML = '';
+  containerOut.appendChild(navCont);
+}
+
+// navigation();
+export { navbar as default };
```
Comment (bertil291utn) [inline src/js/navbar.js]: ### Inline comments
- Remove inline comments, it's not good practice to leave code

File: src/js/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import homePage from './page-load';
+import menu from './menu';
+import navbar from './navbar';
+import contact from './contact';
+
+require('../css/style.css');
+
```
Comment (bertil291utn) [inline src/js/index.js]: ### Use ES6 syntax
- [x] Use import for CSS files and remember to have a configuration for CSS files loader in webpack file

**Review by bertil291utn — CHANGES_REQUESTED** [review]
# **Status:** Changes requested 🟠 
<p align="center"> image <br><strong>Not yet</strong> </p>

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing.

However, in order to accomplish the actual page, you required to do these changes, listed below:
### General requirements
- Next projects, please don't; forget to add a commit message after a function or something relatable that you could mention in the commit message because it's not good practice to commit at the end of the project. Also when you have recurrent commit you could use a wide range to use git tool (rebase, merge, rollback some error)
- [x] Add acknowledge section (if you have something ), giving credits to the inspirations page, or where you get information to build this project
- [x] "Built with" section. Add links to each technology. Because when someone arrives at the README file it'd be great to know where to find this technology. Exampleimage
* [x] Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open"
  },
```
- [x] It appends HTML elements to the `<div>` with `id=content` when the page is loaded (Use `document.createElement` and `document.appendChild` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))
  - Add an `index.html` file then use as root pointer to start all page, follow this [example](https://github.com/bertil291utn/restaurant-page) 


### OPTIONAL
- It'd be great to don't have the home link section in the navbar element, because when you land for the first time to the page you have already the home, and to return to the home page just clicking on the logo 
- Add the logo in the navbar section as a link to redirect to the index page 
- It'd be great to have on the home section an introduction. This page's great in that it should go to your portfolio, and to do that we need to add great information to display this page 
- Same for other pages add real information and actual images 
- Give a style to the form on the 'contact us' section 
- Use `export default <your object/function>` for all modules export,  because it's more readable 

### Summary
- Add more information, and real images and also update styling in the contact form element, because this project could be on your portfolio 
- Use ES6 syntax for import
- Add `index.html` to start as root file. Remember to add webpack configuration, pointing to the new index.html file, following the example 
- Add scripts to start and build its project
-  Don't forget to work with inline comments below this comment box.

***

Guys work on the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**Good work so far here. You did quite well here fixing most issues raised in the previous review.** 👍

However, let's make a few more fixes and we are good to go.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+    "name": "the-restaurant-page",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "dev": "webpack --mode development",
+        "build": "webpack --mode production",
+        "start": "webpack-dev-server --mode development --open"
```
Comment (mikenath223) [inline package.json:9]: - [x] If the webpack dev server option is being used please include the `webpack-dev-server` package in the project.

File: src/css/style.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+.logo {
+  display: block;
+  width: 70px;
+  height: 70px;
+}
+
+.foodContainer {
+  width: 20rem;
+  max-height: 300px !important;
```
Comment (mikenath223) [inline src/css/style.css]: - [x] Kindly avoid using the !important rule in css as it doesn't meet the best practices requirements. Please remove it.

File: src/js/contact.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
+function contact() {
+  const containerOut = document.getElementById('content');
+
+  const h1 = document.createElement('h1');
+  const h1Text = document.createTextNode('Restaurant de Bon Appetite');
+  h1.appendChild(h1Text);
```
Comment (mikenath223) [inline src/js/contact.js]: - [x] Please change this function to es6 arrow function syntax.

File: src/js/menu.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import img from '../burger.jpg';
+
+
+function menu() {
+  const containerOut = document.getElementById('content');
+  const foodItems = document.createElement('div');
+  foodItems.classList.add('food-items');
+  for (let i = 0; i < 4; i += 1) {
+    const foodContainer = document.createElement('div');
```
Comment (mikenath223) [inline src/js/menu.js]: - [x] Also kindly change this function to es6 arrow function syntax too.

File: src/js/navbar.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
+import img from '../logo.jpg';
+
+
+function navbar() {
+  const navCont = document.createElement('div');
+  const imageCont = document.createElement('div');
+
+  const tabsCont = document.createElement('div');
```
Comment (mikenath223) [inline src/js/navbar.js]: - [x] Also change this function to es6 arrow function syntax too.

File: src/js/page-load.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
+function homePage() {
+  const containerOut = document.getElementById('content');
+  const h1 = document.createElement('h1');
+  const h1Text = document.createTextNode('Restaurant de Bon Appetite');
+  h1.appendChild(h1Text);
```
Comment (mikenath223) [inline src/js/page-load.js]: - [x] Also change this function to es6 arrow function syntax.

**Review by whiz25 — APPROVED** [review]
## Approved :white_check_mark:
Good job guys. You changed all functions in the src folder to ES6 arrow functions as requested. Your codebase now looks clean and modern.
Please merge and proceed to the next milestone.


### Martin Afani

**Pull request merged**